widget/list.go

```diff
@@ -274,6 +290,16 @@ func (l *List) ScrollToOffset(offset float32) {
 	l.Refresh()
 }
 
+// ScrollUpOnePage scrolls up one page (table height)
+func (l *List) ScrollUpOnePage() {
```

Comment:
I'm not sure these need to be public APIs (you can do the equivalent by `ScrollToOffset(l.GetScrollOffset() + l.Size().Height)`) but if they are going to be new public APIs they need the `// Since: 2.5` designation